src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateExecution.java

```diff
@@ -225,8 +226,8 @@ private String computeDefaultPathOnDisk(@Nonnull String diskId, @CheckForNull St
             throw new AbortException(message);
         }
 
-        FilePath diskFilePath = new FilePath(new File(physicalPathOnDisk));
-        return new FilePath(diskFilePath, run.getParent().getFullName() + '/' + run.getNumber()).getRemote();
+        File pathOnDisk = Paths.get(physicalPathOnDisk, run.getParent().getFullName(), String.valueOf(run.getNumber())).toFile();
```

Comment:
I did another change here.
There wasn't any bug, but Instead of concatenating the file path with the separator, I think it's safer to use the Java APIs for this.